website/docs/docs/collaborate/trust-signals.md

```diff
@@ -0,0 +1,87 @@
+
+Trust signal icons offer a quick, at-a-glance view of data health when browsing your resources in dbt Explorer. These icons keep you informed on the status of your resource's health using the indicators **Healthy**, **Caution**, **Degraded**, or **Unknown**.
+
+- Supported resources are [models](/docs/build/models), [sources](/docs/build/sources), and [exposures](/docs/build/exposures).
+- For accurate health data, ensure the resource is up-to-date and had a recent job run.
+- Each trust signal icon reflects key data health components, such as test success status, missing resource descriptions, missing tests, absence of builds in 30-day windows, [and more](#trust-signal-criteria)
+
+<Lightbox src="/img/docs/collaborate/dbt-explorer/trust-signal-health.jpg" width="55%" title="View trust signals for your models."/> 
+
+## Access trust signal icons
+
+Access trust signal icons in the following places:
+- [Search function](/docs/collaborate/explore-projects#search-resources) or click on **Models**, **Sources** or **Exposures** under the **Resource** tab. 
+   - For sources, the trust signal also indicates the source freshness status.
+- View the icons under the **Health** column in [each resource's details page](#view-resource-details). Hover over or click the trust signal to view detailed information.
+- Public models tables under the **Health** column.
+- DAG lineage graph 
+
+
+<Lightbox src="/img/docs/collaborate/dbt-explorer/icons-trust-signal.gif" width="95%" title="Access trust signal icons in multiple places in dbt Explorer."/> 
+
+## Trust signal criteria
+
+Each resource has a health state that is determined by specific set of criteria. Select the following tabs to view the criteria for that resource type.
+<Tabs>
+<TabItem value="models" label="Models">
+
+The health state of a model is determined by the following criteria:
+
+| **Health state** | **Criteria**   |
+|-------------------|---------------|
+| ✅ **Healthy**    | - Built successfully in the last run AND<br />- Built in the last 30 days AND<br />- Model has tests configured AND<br />- All tests passed AND<br />- All upstream sources are fresh or freshness is not applicable (set to `null`) AND<br />- Has a description |
+| 🟡 **Caution**   | - Not built in the last 30 days OR<br />- Tests are not configured OR<br />- Tests return warnings OR<br />- One or more upstream sources are stale:<br />&nbsp;&nbsp;&nbsp;&nbsp;- Has a freshness check configured<br />&nbsp;&nbsp;&nbsp;&nbsp;- Freshness check ran in the past 30 days<br />&nbsp;&nbsp;&nbsp;&nbsp;- Freshness check returned a warning<br />- Missing a description |
+| 🔴 **Degraded**  | - Model failed to build OR<br />- Model has failing tests OR<br />- One or more upstream sources are stale:<br />&nbsp;&nbsp;&nbsp;&nbsp;- Freshness check hasn’t run in the past 30 days<br />&nbsp;&nbsp;&nbsp;&nbsp;- Freshness check returned an error |
+| ⚪ **Unknown**    | - Unable to determine health of resource; ingestion hasn't run           |
```

Comment:
for 'ingestion hasn't run' -- do we mean a job run hasn't happened?

Reply:
jarrod might be better suited for this one. My understanding is that the resource hasn't gone through any runs

Reply:
Unable to determine health of resource; no job runs have processed the resource.

Reply:
Yes, your understanding is correct and that suggested text makes sense!